blueprint/api.apib

```diff
@@ -2174,6 +2315,30 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number)
++ created: `2016-07-08T03:03:51.967Z` (string)
++ name: `Emerald Personal` (string)
```

Comment:
I think this is missing `inserted-at` and `updated-at` timestamps, which actually all objects in our database will have.